src/Common/ExtensionMethods.cs

```diff
@@ -23,7 +23,7 @@ namespace Nvidia.Clara.DicomAdapter.Common
 {
     public static class ExtensionMethods
     {
-        public const int CLARA_JOB_NAME_MAX_LENGTH = 25;
+        public const int CLARA_JOB_NAME_MAX_LENGTH = 253;
```

Comment:
I thought the limitation was 15 characters. The tight limit is enforced by any job which uses Argo as its orchestration engine. Not sure if DA wants to be in the business of determining which orchestrator will be used.

Reply:
The max length of job name should come from the job API, shouldn't it? DICOM Adaptor, as far as I can tell, does not expose the job name to its clients, and creates the job name internally itself. It should get the max length from its dependency.

Reply:
job name length limit is imposed by Argo, and I cannot for the life of me remember its restrictions but they were pretty tight.